cfme/utils/template/rhopenshift.py

```diff
@@ -138,6 +138,10 @@ def run(self):
             logger.info("%s:%s No podtesting tag.", self.log_name, self.provider_key)
             return
 
+        if '511' in self.stream or 'upstream' in self.stream:
```

Comment:
is this a typo, is it not 5.11?

Reply:
it looks like downstream-511z

Reply:
exactly, the stream doesn't have a dot in it. We either have to use `template_name` or `stream` here, and I chose `stream`.